newsfragments/2632.feature

```diff
@@ -0,0 +1 @@
+``dials.correlation_matrix``: Correlation and cos-angle clustering of multi-crystal datasets independent of xia2.multiplex. HTML output included of clustering heatmaps, dendrograms and corresponding cosym graphs.
```

Comment:
Since it's a new feature, I'd suggest making a slightly bigger deal of it in the newsfragment.  Something a bit like
```suggestion
``dials.correlation_matrix``: A new command-line tool for correlation and cosine similarity clustering of multi-crystal datasets, independent of ``xia2.multiplex``. It provides HTML output, including clustering heatmaps, dendrograms and corresponding ``dials.cosym`` graphs.
```